components/server/src/github/github-repository-provider.ts

```diff
@@ -171,13 +171,14 @@ export class GithubRepositoryProvider implements RepositoryProvider {
         }
     }
 
-    async getUserRepos(user: User): Promise<string[]> {
+    async getUserRepos(user: User): Promise<RepositoryInfo[]> {
         const result: any = await this.githubQueryApi.runQuery(
             user,
             `
             fragment Repos on RepositoryConnection {
                 nodes {
                   url
+                  name
```

Comment:
Including `name` in the results we return for getUserRepos() to include in response